homeassistant/components/netio/switch.py

```diff
@@ -29,8 +32,8 @@
 
 DEFAULT_PORT = 1234
 DEFAULT_USERNAME = "admin"
-Device = namedtuple("device", ["netio", "entities"])
-DEVICES = {}
+Device = namedtuple("Device", ["netio", "entities"])
```

Comment:
Why did this string change?

Reply:
Because mypy demands that the key of namedtuple is the same as the receiving variable.

Reply:
oh right, the type name 🤦